$MirOS: ports/infrastructure/install/patch-bsd_sys_mk,v 1.1.7.1 2005/03/18 15:47:16 tg Exp $

This patch is necessary for using <bsd.port.mk> and
<bsd.lib.mk> within MirPorts.

Applies to (at least):
* OPENBSD_2_8
* OPENBSD_2_9
* OPENBSD_3_0
* OPENBSD_3_1
* OPENBSD_3_2
* OPENBSD_3_3
* OPENBSD_3_4
* OPENBSD_3_5
* OPENBSD_3_6
* OPENBSD_3_7
* MIRBSD_7ter
* MIRBSD_7quater

Not needed when using portable MirMake with MirPorts.

Index: bsd.sys.mk
===================================================================
RCS file: /cvs/src/share/mk/bsd.sys.mk,v
retrieving revision 1.8
diff -d -u -r1.8 bsd.sys.mk
--- bsd.sys.mk	2000/07/06 23:12:41	1.8
+++ bsd.sys.mk	2004/06/15 14:44:53
@@ -3,6 +3,9 @@
 #
 # Overrides used for OpenBSD source tree builds.
 
+# The following is only wanted for source tree builds, not MirPorts
+.ifndef TRUEPREFIX
+
 #CFLAGS+= -Werror
 
 .if defined(DESTDIR)
@@ -36,3 +39,5 @@
 	${COMPILE.c} -o ${.TARGET} ${.TARGET:R}.tab.c
 	rm -f ${.TARGET:R}.tab.c
 .endif
+
+.endif # ndef TRUEPREFIX
